examples/RetrieveAuditEvents.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2016, 2018, Oracle and/or its affiliates. All rights reserved.
+'''
+This script retrieves all audit logs across an OCI Tenancy
+for a defined timespan.
+
+This script relies on OCI config as per the format defined at
```

Comment:
I feel that lines 8-17, apart from line 11, are superfluous and we don't need to include in every script what the prerequisites for using the SDK are as those are documented elsewhere.